src/DeepFace.php

```diff
@@ -300,6 +336,21 @@ protected function run(string $filepath, array $data): array|bool
             ->mustRun()
             ->getOutput();
 
+        $errorOutput = $process->getErrorOutput();
+
+        if(!empty($errorOutput)){
+            if (preg_match_all('/\{(?:[^{}]|(?R))*\}/', $errorOutput, $matches)) {
+                $lastJson = end($matches[0]);
+                $errorResult = json_decode($lastJson, true);
+
+                if ($errorResult !== null && isset($errorResult['error'])) {
+                    throw new DeepFaceException($errorResult['error']); // should return 'Spoof detected in the given image'
```

Comment:
I left it there since I thought that Deepface returned "Spoof detected in the given image" everytime, but upon inspection of the recent changes they added, it returns a different message under `ValueError`, so, `$errorResult['error']` can be any error thrown by the script.

I can remove the comment if you want.

Reply:
Done